### PR TITLE
more chs translation

### PR DESCRIPTION
**Pull request by duskgao**
RT

### CODE REVIEW

File: intl/msg_hash_chs.c [hunk 1]
```diff
@@ -67,19 +67,19 @@ int menu_hash_get_help_chs_enum(enum msg_hash_enums msg, char *s, size_t len)
...
             break;
          case RARCH_SCREENSHOT:
             snprintf(s, len,
-                  "Take screenshot.");
+                  "截图.");
             break;
          case RARCH_MUTE:
             snprintf(s, len,
-                  "Mute/unmute audio.");
+                  "静音/取消经营.");
```
Comment (UlyssesWu) [inline intl/msg_hash_chs.c]: 错别字！:(

File: intl/msg_hash_chs.c [hunk 2]
```diff
@@ -112,11 +112,11 @@ int menu_hash_get_help_chs_enum(enum msg_hash_enums msg, char *s, size_t len)
             break;
          case RARCH_VOLUME_UP:
             snprintf(s, len,
-                  "Increases audio volume.");
+                  "调高音量.");
```
Comment (UlyssesWu) [inline intl/msg_hash_chs.c]: “调高”与“降低”对应不佳。提高-降低；增大-减小

**Comment by twinaphex** [comment]
@UlyssesWu What's the consensus on this? Can this be merged or not?

**Comment by UlyssesWu** [comment]
@twinaphex Basically the translations are correct, but with 2 typos. The first one is obvious and a little confusing. 
I hope he could fix them before this is merged.

**Comment by twinaphex** [comment]
@UlyssesWu Would it work also that we merge this PR and then later on you can correct these two strings in a followup PR, or would you prefer he do it instead?

**Comment by UlyssesWu** [comment]
@twinaphex I think my friend @9chu (who have contributed a lot in this translation) would like to correct them later. So it's ok to merge it now if needed.

**Comment by twinaphex** [comment]
@UlyssesWu OK, I will merge it now then. Thanks.

**Pull request merged**